calibrators/marker_radar_lidar_calibrator/src/marker_radar_lidar_calibrator.cpp

```diff
@@ -459,19 +520,37 @@ void ExtrinsicReflectorBasedCalibrator::lidarCallback(
     converged_tracks_.size());
 }
 
-void ExtrinsicReflectorBasedCalibrator::radarCallback(
+void ExtrinsicReflectorBasedCalibrator::radarTracksCallback(
   const radar_msgs::msg::RadarTracks::SharedPtr msg)
 {
-  if (!latest_radar_msgs_) {
-    latest_radar_msgs_ = msg;
+  if (!latest_radar_tracks_msgs_) {
```

Comment:
Regarding the methods `radarTrackscallback` and `radarScanCallback`.

At the beginning of X2 Gen1, we has driver issues and the frequency of the radar was dozen of times higher than the lidars.

This caused the need to aggregate the radar scans and make the calibration radar-centric (i.e., calibrate each lidar scan).

This error was fixed somewhere along the line and now the rosbags look like this:
```
Topic information: Topic: /sensing/lidar/front_lower/pointcloud_raw | Type: sensor_msgs/msg/PointCloud2 | Count: 2064 | Serialization Format: cdr
                   Topic: /sensing/radar/front_center/objects_raw | Type: radar_msgs/msg/RadarTracks | Count: 2738 | Serialization Format: cdr
                   Topic: /tf_static | Type: tf2_msgs/msg/TFMessage | Count: 1 | Serialization Format: cdr
```

i.e., almost the same frec.
I think we can safely remove this approach now

Reply:
@knzo25 Do you mean like this?
https://github.com/tier4/CalibrationTools/pull/180/commits/8ab4a0c2fffa75abdd7d8ec0d921748dfa2bc85f

Reply:
fix: https://github.com/tier4/CalibrationTools/pull/180/commits/8ab4a0c2fffa75abdd7d8ec0d921748dfa2bc85f
revert: https://github.com/tier4/CalibrationTools/pull/180/commits/b6c3c69c4076575c97d61f3da5d7bae5ae4c1567

I revert this fix since it causes a problem on x2 gen2 calibration.
I will fix this later.